packages/location-field/src/options.tsx

```diff
@@ -127,36 +158,30 @@ export function UserLocationIcon({
   return <MapMarker size={ICON_SIZE} />;
 }
 
-function LocationName({ location }: { location: UserLocation }): ReactElement {
+function getLocationName(location: UserLocation, intl: IntlShape): string {
   switch (location.type) {
     case "home":
-      return (
-        <FormattedMessage
-          defaultMessage="Home"
-          description="The home location"
-          id="otpUi.LocationField.homeLocation"
-        />
-      );
+      return intl.formatMessage({
+        defaultMessage: "Home",
+        description: "The home location",
+        id: "otpUi.LocationField.homeLocation"
+      });
     case "work":
-      return (
-        <FormattedMessage
-          defaultMessage="Work"
-          description="The work location"
-          id="otpUi.LocationField.workLocation"
-        />
-      );
+      return intl.formatMessage({
+        defaultMessage: "Work",
+        description: "The work location",
+        id: "otpUi.LocationField.workLocation"
+      });
     default:
-      return <>{location.name}</>;
+      return location.name;
```

Comment:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/86619099/234367090-5ba22f09-dcf9-4a16-96a0-88d7366873b6.png">
Not sure if this is related to this line, but I'm seeing this on the HERE story!

Reply:
This is a current behavior of the component. What was the desired effect?

Reply:
It looks that the mock is missing one field. I don't remember if that was done on purpose or not. Please advise.

Reply:
Oh sorry about that I think I got confused. Yes I think it's intentional to check if there's a crash